### PR TITLE
Tests:  Automation of two bz

### PR DESCRIPTION
**Pull request by aborah-sudo**
https://bugzilla.redhat.com/show_bug.cgi?id=1986782
https://bugzilla.redhat.com/show_bug.cgi?id=1989900

### CODE REVIEW

File: multihost_test/bz_automation/conftest.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""conftest.py for authselect"""
```
Comment (ikerexxe) [inline multihost_test/bz_automation/conftest.py:1]: The code in this file is almost the same as in `multihost_test/Sanity/conftest.py`. Can't you reuse it?

File: multihost_test/bz_automation/test_bz_automation.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+
+import pytest
+import subprocess
+import os
+import time
+
+
+def exceute_cmd(multihost, command):
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_bz_automation.py:8]: s/exceute_cmd/execute_cmd

File: multihost_test/bz_automation/test_bz_automation.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        """
+        :title: Groupdel gives segmentation fault when using '-P'
+        :id: f86daef6-540c-11ec-a7c2-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1986782
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2024834
+        """
+        for i in ["mkdir -p $HOME/custom_root/etc/",
+                  "touch $HOME/custom_root/etc/group",
+                  "groupadd -P $HOME/custom_root test_group",
+                  "groupdel -P $HOME/custom_root test_group"]:
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_bz_automation.py:25]: I think that you should check the return value for this command.

File: multihost_test/bz_automation/test_bz_automation.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        :title: For nofile, a value of "unlimited" should be allowed.
+        :id: 54d940a6-540d-11ec-9a01-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1989900
+        """
+        exceute_cmd(multihost, "cp -vf /etc/security/limits.conf "
+                               "/etc/security/limits.conf_anuj")
+        exceute_cmd(multihost, "useradd anuj_test")
+        exceute_cmd(multihost, 'echo "anuj_test hard nofile -1"'
+                               ' >> "/etc/security/limits.conf"')
+        exceute_cmd(multihost, "su -c 'id' anuj_test")
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_bz_automation.py:39]: Again, you should check the return value for this command.

File: multihost_test/bz_automation/test_bz_automation.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                  "groupdel -P $HOME/custom_root test_group"]:
+            exceute_cmd(multihost, i)
+
+    def test_unlimited(self, multihost):
+        """
+        :title: For nofile, a value of "unlimited" should be allowed.
+        :id: 54d940a6-540d-11ec-9a01-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1989900
+        """
+        exceute_cmd(multihost, "cp -vf /etc/security/limits.conf "
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_bz_automation.py:34]: I don't think it's a good idea to backup to `/etc` folder. Can you do it for example in `/tmp`?

**Review by ikerexxe — APPROVED** [review]
The changes overall seem good but there are some things to change.

**Review by ikerexxe — CHANGES_REQUESTED** [review]
I set `Approve` instead of `Request changes` by mistake.

File: multihost_test/bz_automation/test_bz_automation.py (hunk 4 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/test_bz_automation.py:39]: the test will fail if this command fails . So no need to check the output separately

File: multihost_test/bz_automation/test_bz_automation.py (hunk 3 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/test_bz_automation.py:25]: the test will fail if this command fails . So no need to check the output separately

File: multihost_test/bz_automation/test_bz_automation.py (hunk 2 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/test_bz_automation.py:8]: done

File: multihost_test/bz_automation/test_bz_automation.py (hunk 5 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/test_bz_automation.py:34]: done

File: multihost_test/bz_automation/conftest.py (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/conftest.py:1]: that way IDM CI does not work , you can see sssd repo for the same , every test directly has its own conftest.py file

File: multihost_test/bz_automation/test_bz_automation.py (hunk 3 above)
Comment (ikerexxe) [reply multihost_test/bz_automation/test_bz_automation.py]: Ok. Makes sense.

**Comment by ikerexxe** [comment]
Just one thing before I approve it. One of the tests seems to have banished. What did it happen? Are you going to fill a separate PR for it?

**Comment by aborah-sudo** [comment]
> Just one thing before I approve it. One of the tests seems to have banished. What did it happen? Are you going to fill a separate PR for it?

I have moved one test to shadow-utils repo . As was wrongly put here with this PR . 

https://github.com/RedHatQE/shadow-utils/pull/6

**Review by ikerexxe — APPROVED** [review]

**Pull request merged**